Tasks/AppCenterDistributeV2/appcenterdistribute.ts

```diff
@@ -369,6 +391,41 @@ function expandSymbolsPaths(symbolsType: string, pattern: string, continueOnErro
     return symbolsPaths;
 }
 
+function getReleaseId(apiServer: string, apiVersion: string, appSlug: string, releaseId: string, token: string, userAgent: string): Q.Promise<any> {
+    tl.debug("-- Getting release id.");
+    let defer = Q.defer<any>();
+    let getReleaseUrl: string = `${apiServer}/${apiVersion}/apps/${appSlug}/uploads/releases/${releaseId}`;
+    tl.debug(`---- url: ${getReleaseUrl}`);
+    let headers = {
+        "X-API-Token": token,
+        "User-Agent": userAgent,
+        "internal-request-source": "VSTS"
```

Comment:
It's still VSTS? (not AzDO)

Reply:
With this task (AzDO) I receive this error: 
`Cannot find module 'AzDO-task-lib/task' or its corresponding type declarations.`